lib/src/api/opt/query.rs

```diff
@@ -160,19 +152,19 @@ impl IntoQuery for OptionStatement {
 
 impl IntoQuery for &str {
 	fn into_query(self) -> Result<Vec<Statement>> {
-		sql::parse(self)?.into_query()
+		syn::parse(self)?.into_query()
```

Comment:
I wouldn't expect this change to be necessary. Does this mean `surrealdb::sql::parse` is no longer available @DelSkayn?